pkg/vendir/fetch/ref_fetcher.go

```diff
@@ -27,6 +27,6 @@ func (f SingleSecretRefFetcher) GetSecret(name string) (ctlconf.Secret, error) {
 	return ctlconf.Secret{}, fmt.Errorf("Not found")
 }
 
-func (f SingleSecretRefFetcher) GetConfigMap(name string) (ctlconf.ConfigMap, error) {
+func (f SingleSecretRefFetcher) GetConfigMap(_ string) (ctlconf.ConfigMap, error) {
```

Comment:
If we are not using this param here then why we are not removing it instead of using `_`?

Reply:
Here `GetConfigMap` is part of `RefFetcher` interface and we can't change the method signatures.